projects/plugins/boost/tests/e2e/specs/page-cache/page-cache.test.js

```diff
@@ -12,6 +12,7 @@ test.describe( 'Cache module', () => {
 	test.beforeAll( async ( { browser } ) => {
 		page = await browser.newPage( playwrightConfig.use );
 		await boostPrerequisitesBuilder( page )
+			.withLoggedIn( true )
```

Comment:
I am assuming this is to make sure the page cache setup e2e test suit doesn't need to depend on another suite for a logged in state. Am I correct?

Reply:
Yep. While testing individually I ran into a problem where the test was failing because the test was logged out 😅